index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="EN">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@700&family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap"
+     rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Poppins:wght@100;200;400&family=Roboto:wght@100;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Crete+Round&family=El+Messiri&family=Inter&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap" 
+    rel="stylesheet">
+    <title>Riyaz's Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="site-header">
+        <div id="title-text"><h3>
+          <a class="logo-text" href="#">Riyaz Ahmed</a>
+        </h3>
+      </div>
+        <nav id="navbar">
+          <a class="nav-links" href="#">Portfolio</a>
+          <a class="nav-links" href="#">About</a>
+          <a class="nav-links" href="#">Contact</a>
+          <a><i id="menu" class="material-icons">mail</i></a>
+        </nav>
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
```

Comment:
I think you missed the previous reviewer's requirement. According to the evaluation guidelines, the hamburger itself should be wrapped inside a `<nav>` tag. Kindly fix this.

<img width="373" alt="Screen Shot 2022-05-26 at 8 28 31" src="https://user-images.githubusercontent.com/82426682/170497250-889bb84d-d9d9-4641-b92e-78e710638e65.png">

Reply:
I HAVE DONE THAT , and has nav tag 

 <nav class="hamburger">
        <a href="javascript:void(0);" class="icon">
          <i class="fa fa-bars"></i>
        </a>
        </nav>

Reply:
It is having nav tag